<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>800</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>100</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="coverButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="singleButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QToolButton" name="moreButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>160</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="settingButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>720</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>0</y>
      <width>250</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="searchEdit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>30</y>
       <width>220</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="friendList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>250</width>
       <height>710</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>0</y>
      <width>825</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>620</y>
       <width>825</width>
       <height>180</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>825</width>
       <height>590</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="emojiButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>590</y>
       <width>47</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="picButton">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>590</y>
       <width>47</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>750</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
